selectors.yml

```diff
@@ -207,6 +207,27 @@ selectors:
   2021:
     selector: 13116810400804392105
     name: "ronin-testnet-saigon"
+  200810:
+    selector: 3789623672476206327
+    name: "bitcoin-testnet-bitlayer-1"
+  808813:
+    selector: 5535534526963509396
+    name: "ethereum-testnet-sepolia-bob-1"
+  37111:
+    selector: 6827576821754315911
+    name: "ethereum-testnet-sepolia-lens-1"
```

Comment:
I couldn't find much info about lens testnet, it makes sense for it to be an L2 but can you provide a reference

Reply:
yes, from here https://lens-network-alpha.vercel.app/docs/network/overview#differences-with-ethereum:
_The Lens Network is a Layer 2 blockchain network built on [ZKsync](https://zksync.io/) technology. As a result, the Lens Network is EVM-compatible, meaning it shares ZKSync's differences from Ethereum and other EVM-equivalent blockchains._

Reply:
as per this description 
`<blockchain>-<type>-<network_name>-<parachain>-<rollup>-<rollup_instance>` wouldn't it be best to do `ethereum-testnet-sepolia-lens-zksync-1` ?

Reply:
Zksync seems to be only the sdk that the network uses but it doesn't settle on it. See this [thread](https://chainlink.enterprise.slack.com/archives/C04Q61M2MFT/p1728053738994389?thread_ts=1728008968.911349&cid=C04Q61M2MFT) for a similar discussion on why we drop the protocol/sdk.